src/actions/analytics.ts

```diff
@@ -74,10 +74,14 @@ export const trackScreenChange = (
 
   FBAnalytics().setUserProperties({
     debug: __DEV__.toString(),
-    cru_grmasterpersonid: analytics[ANALYTICS_GR_MASTER_PERSON_ID],
+    cru_grmasterpersonid: analytics[ANALYTICS_GR_MASTER_PERSON_ID]
+      ? analytics[ANALYTICS_GR_MASTER_PERSON_ID]
+      : null,
```

Comment:
I think it's better not to duplicate these. It's one more place for a typo.
```suggestion
    cru_grmasterpersonid: analytics[ANALYTICS_GR_MASTER_PERSON_ID] || null,
```